### PR TITLE
Feedback buttons not visible

### LINKED ISSUE #276: Feedback - buttons are not visible
Ticket:

https://projects.bcc.no/desk/tickets/6931027/messages

Buttons are cut off & couldn't sent feedback
image

**Acceptance Criteria**
1. 

root cause: 5455ca8446bc78f3c8dad44bffbe70ee504912b7

### PR DESCRIPTION
**Pull request by JakubC-projects**
closes #276 

This does few things:
1. I disable scroll on the whole document when popover is opened, so that the scroll in the popover works more predictably.
2. I fix the star bug (on iOS the stars had big padding at the top and at the bottom)
3. I add big padding at the bottom of the popover so that mobile devices can actually scroll down to see the buttons

### CODE REVIEW

File: client/src/components/feedback-popover.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
             <div class="mx-auto" style="max-width:386px">
                 <h5 class="mb-2">{{$t('labels.rate-experience')}}</h5>
                 <div class="flex justify-between mb-5">
-                    <StarIcon v-for="i in maxRating" :key="i" class="w-12" @click="rating = i" :style="getColor(i)"/>
+                    <StarIcon v-for="i in maxRating" :key="i" class="w-12 h-12" @click="rating = i" :style="getColor(i)"/>
                 </div>
                 <FormField class="mb-9" type="textarea" translation="any-trouble" v-model="textVal" baseHeight="100px" :placeholder="$t('placeholders.type-here')" enableNewLine/>
```
Comment (StevenMalaihollo) [inline client/src/components/feedback-popover.vue:14]: Feedback check should look at something other than NODE_ENV

File: client/src/components/popover-base.vue [hunk 2]
```diff
@@ -13,6 +13,12 @@ export default defineComponent({
     props: {
         mobileHeight: {type: String, default: "50vh"}
     },
+    mounted() {
+        document.documentElement.style.overflow = 'hidden'
+    },
+    unmounted() {
```
Comment (StevenMalaihollo) [inline client/src/components/popover-base.vue:19]: Look for an alternate solution for overflow scrolling

File: client/src/components/feedback-popover.vue (hunk 1 above)
Comment (JakubC-projects) [reply client/src/components/feedback-popover.vue:14]: done

File: client/src/components/popover-base.vue (hunk 2 above)
Comment (JakubC-projects) [reply client/src/components/popover-base.vue]: I just removed that logic

**Review by StevenMalaihollo — APPROVED** [review]

**Pull request merged**